java/operator/src/main/java/com/instaclustr/cassandra/operator/controller/DataCenterDeletionController.java

```diff
@@ -29,11 +29,11 @@ public DataCenterDeletionController(final CoreV1Api coreApi,
     public void deleteDataCenter() throws Exception {
         final String labelSelector = String.format("cassandra-datacenter=%s", dataCenterKey.name);
 
-        // delete persistent volumes & persistent volume claims
-        final V1PodList pods = coreApi.listNamespacedPod(dataCenterKey.namespace, null, null, null, null, labelSelector, null, null, null, null);
```

Comment:
why don't we want to delete the pvc claim? Is this done by the stateful set?

Reply:
Will test.